lib/mocha.js

```diff
@@ -1219,22 +1221,40 @@ Mocha.prototype.runGlobalTeardown = async function runGlobalTeardown(
 };
 
 /**
- * Run global fixtures sequentially with context `context`
+ * Run global fixtures sequentially with context `context`.
```

Comment:
Still some unrelated changes: a blank line above, this `.`, an extra `,` later, etc. - https://www.joshuakgoldberg.com/blog/split-out-unrelated-changes -> https://www.joshuakgoldberg.com/blog/split-out-unrelated-changes/#separate-logical-and-stylistic-changes

Reply:
@TG199 this piece of feedback is still open